utils/build-windows-toolchain.bat

```diff
@@ -855,8 +855,9 @@ endlocal
 :FetchWiX
 setlocal enableextensions enabledelayedexpansion
 
+if exist wix-4.0.1 goto :eof
```

Comment:
what is `:eof`? Don't we want to just finish this function and keep going with the toolchain build?

Reply:
Oh, eof, "end-of-function", not "end-of-file".
Have to love common terms being abbreviated to the same thing while meaning different things.